src/login/logind-core.c

```diff
@@ -396,23 +396,28 @@ int manager_get_user_by_pid(Manager *m, pid_t pid, User **ret) {
         return !!u;
 }
 
-int manager_get_idle_hint(Manager *m, dual_timestamp *t) {
+int manager_get_idle_hint(Manager *m, dual_timestamp *since, dual_timestamp *inactive_since) {
```

Comment:
our coding style these days sugests naming return parameters ret_xyz. i.e. please call the two timestamps ret_idle_since and ret_inactive_since or so